web/src/main/java/org/cbioportal/genome_nexus/web/EnsemblController.java

```diff
@@ -129,7 +129,7 @@ public EnsemblController(EnsemblService ensemblService)
     }
 
     @ApiOperation(value = "Retrieves Ensembl transcripts by an Ensembl gene ID",
-        nickname = "fetchEnsemblTranscriptsByGeneIdGET")
+        nickname = "fetchCanonicalEnsemblTranscriptsByGeneIdGET")
```

Comment:
Also rename the method name to `fetchCanonicalEnsemblTranscriptsByGeneIdGET`?

Reply:
good catch

Reply:
@onursumer updated